### PR TITLE
Release/v2.13.0

### PR DESCRIPTION
**Pull request by carpawell**
Blocked by #225.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## [2.13.0] - 2022-06-21 - Yeonpyeongdo (연평도, 延坪島)
+
+### Added
+- EXtended headers usage clarification (#204)
```
Comment (fyrchik) [inline CHANGELOG.md:6]: s/X/x/

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+- `OUT_OF_RANGE` status code to the `object` section (#208)
+- Disabling homomorphic hashing container setting (#217)
+- `LOCK` object behaviour clarification (#221)
+- Storage group members uniqueness constraint (#222)
+- WalletConnect signature type (#206)
```
Comment (fyrchik) [inline CHANGELOG.md:11]: s/type/scheme/

**Review by fyrchik — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (carpawell) [reply CHANGELOG.md:6]: thought that "eXtended header" is good naming. but it looks strange with the first capitalized letter, agree

fixed

File: CHANGELOG.md (hunk 2 above)
Comment (carpawell) [reply CHANGELOG.md:11]: fixed

**Review by realloc — APPROVED** [review]

**Review by realloc — APPROVED** [review]

**Pull request merged**